app/__init__.py

```diff
@@ -0,0 +1,33 @@
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100, hidden: bool = False):
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.alive.append(self)
```

Comment:
Code Efficiency issue: Instead of using `self.alive.append(self)`, use the class name to get access to the class attributes. So it should be `Animal.alive.append(self)`. This is because the attribute `alive` belongs to the class, not the instance.